jackrabbit-jcr-commons/src/main/java/org/apache/jackrabbit/commons/NamespaceHelper.java

```diff
@@ -197,8 +215,7 @@ public String getJcrName(String name)
      */
     public String registerNamespace(String prefix, String uri)
             throws RepositoryException {
-        NamespaceRegistry registry =
-            session.getWorkspace().getNamespaceRegistry();
+        NamespaceRegistry registry = getNamespaceRegistry();
```

Comment:
Any pointers on how much overhead this call is? I was assuming this is not leading to complex and expensive object creations